pupil_sync/pupil_time_sync_follower.py

```diff
@@ -68,10 +68,11 @@ def slew_time(self, offset):
         self.base_offset += offset
 
 
-def run_time_sync_follower(time_fn, jump_fn, slew_fn, pts_group):
```

Comment:
But the actual pts_group name is calculated within the group, just to make sure the spec is followed correctly.